client-hc/src/main/java/com/graphhopper/api/GHMRequest.java

```diff
@@ -146,7 +149,17 @@ public boolean hasPointHints() {
                 this.toPointHints.size() == this.toPoints.size() && !toPoints.isEmpty();
     }
 
-    private int called = 0;
```

Comment:
was this down here intentionally ? sorry if yes

Reply:
in rare situations I need to store state but they are used in just one method, but moving this up is fine :)